rom-hacking/scripting/animation-commands/ac-dir-u.md

```diff
@@ -0,0 +1,30 @@
+# (773) _AC_DIR_U
+
+## Effect
+
+Turns the entity of the animation up over a given amount of frames.
+
+The entity always makes the smallest turn possible. If the **invert** argument is provided, the entity will turn the longer way around.
+
+Starts the stepping sound effect.
+
+## Syntax
+
+```c
+_AC_DIR_U(time, invert?)
+```
+
+| Argument | Description | Types | Required |
+| - | - | - | - |
+| **time** | The amount of time to turn, in frames (1/30ths of a second) | Float | Required |
```

Comment:
I would remove the 1/30ths of a second everywhere it's cited. The game framerate is dependent on user configuration of globalgamemanagers. Not always true.